sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -318,3 +319,14 @@ def test_results_filenames():
     assert len(files) == len(responses)
     for hmiurl in files:
         assert os.path.isfile(hmiurl)
+
+
+@pytest.mark.remote_data
```

Comment:
Rather than making this an online test, you should mock `parfive.download` as we don't actually need to download the files.

Reply:
How should I mock the JSOCResponse object that goes into this,
` client.fetch(responses, path=path, max_conn=5, max_splits=5)`
Hard code one ?

Reply:
yeah, if it's not too much effort you might as well. Even if you end up keeping the remote_data for the search, it's worth not actually doing the download.